pyro/infer/abstract_infer.py

```diff
@@ -278,4 +302,7 @@ def _traces(self, *args, **kwargs):
             yield (replayed_trace, 0., 0)
 
     def marginal(self, sites=None):
+        """
+        Gets marginal distribution from posterior.
+        """
         return self.posterior.marginal(sites)
```

Comment:
not sure why we return marginal of posterior (not predictive) here so I leave it as is.